### PR TITLE
Amy Cham HTML200 Course Project

### PR DESCRIPTION
**Pull request by amybrate**

### CODE REVIEW

File: alldogs.html [hunk 1]
```diff
@@ -0,0 +1,164 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <title>Adoptadog - Seattle's premiere dog adoption agency</title>
+    <link rel="stylesheet" href="css/main.css">
+  </head>
+  <body>
+    <header id='page-header'>
```
Comment (jimmynono) [inline alldogs.html]: Be careful with id's. Generally you want to keep those for use with javascript. It's perfectly acceptable to use a class in this case, even if it's for a single use.

File: alldogs.html [hunk 2]
```diff
@@ -0,0 +1,164 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
```
Comment (jimmynono) [inline alldogs.html:4]: There is a meta tag referenced in the week one slides. You'll want to include that.

File: alldogs.html [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+    <title>Adoptadog - Seattle's premiere dog adoption agency</title>
+    <link rel="stylesheet" href="css/main.css">
+  </head>
+  <body>
+    <header id='page-header'>
+      <a href='#start' class='skiplink'>Skip to content</a>
+      <div id='identity'>
+        <img class='logo' src="images/adoptadog-logo.svg" alt="Adopt a Dog logo">
```
Comment (jimmynono) [inline alldogs.html]: When using quote marks in HTML it's best to use double quotes <div id="identity"> We will reserve single quotes for javascript. It'll matter later on.

File: alldogs.html [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+      </div>
+      <nav>
+          <ul>
+            <li><a href="index.html">Home</a></li>
+            <li><a href="#">Dogs</a></li>
+            <li><a href="blog.html">Blog</a></li>
+          </ul>
+      </nav>
+    </header>
+    <main>
```
Comment (jimmynono) [inline alldogs.html:29]: Love the use of main

File: alldogs.html [hunk 5]
```diff
@@ -0,0 +1,164 @@
...
+      <section>
+        <header>
+          <h1 id='start'>Here are all of our Dogs!</h1>
+        </header>
+        <article class="">
```
Comment (jimmynono) [inline alldogs.html:34]: Be careful to use article tags on things that are actual articles. I'm not sure a wrapping container is the best use of article

File: alldogs.html [hunk 6]
```diff
@@ -0,0 +1,164 @@
...
+          <ul class='cards'>
+            <li class='card'>
+              <img src="images/murphy-card.jpg" alt="Murphy - a large brown dog with floppy ears">
+              <h2>Murphy</h2>
+              <p><strong>Cost to Adopt:</strong> $123.45</p>
```
Comment (jimmynono) [inline alldogs.html]: Thank you for not using the dreaded bold tag!!!

File: alldogs.html [hunk 7]
```diff
@@ -0,0 +1,164 @@
...
+              <p>The least little bit can do so much. Everybody needs a friend.</p>
+              <a href="#" class='action-button'>Adopt</a>
+            </li>
+            <li class='card'>
```
Comment (jimmynono) [inline alldogs.html]: I like the way you lay these out, clean and simple. You MIGHT run into trouble stying the li's, and you might have to rethink how you're grouping these, but give it a shot and hopefully it'll work.

File: alldogs.html [hunk 8]
```diff
@@ -0,0 +1,164 @@
...
+      <img class='logo' src="images/adoptadog-logo.svg" alt="Adopt a Dog logo">
+      <p>Let's build an almighty mountain. We start with a vision in our heart, and we put it on canvas.</p>
+      <nav>
+          <ul>
+            <li><a href="index.html">Home</a></li>
+            <li><a href="#">Dogs</a></li>
+            <li><a href="#">Contact</a></li>
+          </ul>
+      </nav>
+      <p>&copy; 2018 Adoptadog, LLC | Seattle, WA</p>
```
Comment (jimmynono) [inline alldogs.html:136]: Nice use of &copy;

File: blog.html [hunk 9]
```diff
@@ -0,0 +1,80 @@
...
+          <header>
+            <!-- should these non-informational images be article bkg instead?
+                  better for screen readers and layout...? -->
+            <img src="images/blog-2.jpg" alt="Several dogs on leashes sitting on a brick sidewalk">
+            <h2>How to Walk with Multiple Dogs</h2>
+          </header>
+          <p>You gotta think like a tree. Maybe there's a little something happening right here. You've got to learn to fight the temptation to resist these things. Just let them happen. And maybe a little bush lives there. We need a shadow side and a highlight side. See there how easy that is.</p>
+          <p>Have fun with it. Everyone is going to see things differently - and that's the way it should be. Just think about these things in your mind - then bring them into your world.</p>
+        </article>
+        <article class="blog">
```
Comment (jimmynono) [inline blog.html:53]: Nice layout and great use of comments!

File: alldogs.html (hunk 1 above)
Comment (amybrate) [reply alldogs.html]: Haha. Okay. Drupal habit. They slap an id on EVERYTHING!

File: alldogs.html (hunk 3 above)
Comment (amybrate) [reply alldogs.html]: Didn't realize! I'll clean that up as I'm working on the files.

File: alldogs.html (hunk 5 above)
Comment (amybrate) [reply alldogs.html:34]: Ok. I don't need a wrapper, I just included it anticipating that pages will change over time. If in the future there were, say, two FAQ blurbs (which I would likely use article for) added to the end of the page, would we then want the dog list to be an article, too?

File: alldogs.html (hunk 2 above)
Comment (amybrate) [reply alldogs.html:4]: WHOOPS. Fixed.

File: alldogs.html (hunk 5 above)
Comment (jimmynono) [reply alldogs.html:34]: I'd argue that a list of dog tiles isn't semantically an article, but just a wrapping container. I like to use articles on things that are actually articles. A wrapping container is good though, but I just like to keep article elements to articles

File: alldogs.html (hunk 1 above)
Comment (jimmynono) [reply alldogs.html]: Ha!

Did I mention to not use single quotes on things for your attributes in HTML? If not, make sure to use only double quote marks for HTML.

<header id="page-header">

File: css/main.css [hunk 10]
```diff
@@ -1,8 +1,114 @@
...
+  background-color: rgb(255, 255, 255);
+  font-family: var(--body-font);
+}
+
+.skiplink{
+  display: none;
+}
+
+header#page-header{
+  background: var(--main-color) url("../images/header-image.jpg") no-repeat center top;
```
Comment (ClovisI) [inline css/main.css:71]: Awesome way of making this header work with your variable!

**Pull request closed without merging**